### PR TITLE
vocabularies: add funders

### PR DESCRIPTION
**Pull request by alejandromumo**
This PR fixes https://github.com/inveniosoftware/invenio-rdm-records/issues/911

Note: `invenio-vocabularies` dependency's version is set to `0.11.1`, which will be updated from `0.11.0` after https://github.com/inveniosoftware/invenio-vocabularies/issues/147.

### CODE REVIEW

File: tests/resources/vocabularies/test_funders_vocabulary.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    return current_service_registry.get("rdm-funders")
+
+
+@pytest.fixture()
+def example_funder(
+    app, db, es_clear, superuser_identity, funders_service
+):
+    """Example funder."""
+    data = {
+        "id": "cern",
```
Comment (ppanero) [inline tests/resources/vocabularies/test_funders_vocabulary.py:26]: major: we have to modify this with the new `pid` column before merging.

**Review by ppanero — COMMENTED** [review]
Nice work! 🚀 

Just one missing point:
- The `services.schemas.metadata:funding` should change to use the `FunderRelationSchema`

Blockers (nothing to do at the moment)
- We just have to wait to merge/release the vocabulary PR, then adapt the datamodel.

**Review by ppanero — COMMENTED** [review]
Pending on invenio-vocabularies funders merge 🚀

File: setup.cfg [hunk 2]
```diff
@@ -82,11 +83,13 @@ invenio_db.alembic =
 invenio_jsonschemas.schemas =
     invenio_rdm_records = invenio_rdm_records.records.jsonschemas
     affiliations = invenio_vocabularies.contrib.affiliations.jsonschemas
+    funders = invenio_vocabularies.contrib.funders.jsonschemas
```
Comment (ppanero) [inline setup.cfg:86]: missing db models (a bit up)

File: invenio_rdm_records/views.py [hunk 3]
```diff
@@ -26,6 +26,7 @@ def init(state):
     registry.register(
         ext.records_service.draft_files, service_id='rdm-draft-files')
     registry.register(ext.affiliations_service, service_id='rdm-affiliations')
+    registry.register(ext.funders_service, service_id='rdm-funders')
```
Comment (ppanero) [inline invenio_rdm_records/views.py:29]:
```suggestion
    registry.register(ext.funders_service, service_id='funders-service')
```

Comment (ppanero) [reply invenio_rdm_records/views.py:29]: also needs to register indexer

**Comment by slint** [comment]
Included in https://github.com/inveniosoftware/invenio-rdm-records/pull/936

**Comment by ppanero** [comment]
superseeded by https://github.com/inveniosoftware/invenio-rdm-records/pull/936

**Pull request closed without merging**